sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/src/main/resources/application.yaml

```diff
@@ -2,29 +2,29 @@ spring:
   cloud:
     azure:
       eventhub:
-        connection-string: [eventhub-namespace-connection-string]
-        checkpoint-storage-account: [checkpoint-storage-account]
-        checkpoint-access-key: [checkpoint-access-key]
-        checkpoint-container: [checkpoint-container]
+        connection-string: Endpoint=sb://yiehtest.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=H3vdqCIDOarSap8IWoGrXcGZTiHGwjLcfEBL98Ntlkg=
```

Comment:
Recover this file. And delete related resources.

Reply:
Fixed